.github/workflows/testDBChangelog.yml

```diff
@@ -41,5 +41,5 @@ jobs:
         run: touch ../.env && docker compose -f ../docker-compose.yml up -d db
       - name: Run forward migrations
         run: ./gradlew liquibaseUpdate
-      - name: Test rolling back all migrations
-        run: ./gradlew liquibaseRollbackCount -PliquibaseCommandValue=9999
+      - name: Test rolling back on last 20 migrations
+        run: ./gradlew liquibaseRollbackCount -PliquibaseCommandValue=20
```

Comment:
See why this change https://github.com/CDCgov/prime-simplereport/wiki/Database-maintenance#rollback-gotchas